packages/msgraph/test/mockAgent.js

```diff
@@ -2,42 +2,547 @@ import { MockAgent } from 'undici';
 
 const mockAgent = new MockAgent();
 
-const mockPool = mockAgent.get('https://fake.server.com');
+const mockPool = mockAgent.get('https://graph.microsoft.com');
+
+const driveResponse = {
```

Comment:
I would move each fixture data out into its own file, probably in `test/fixtures`. That will really clean up the mock agent

Reply:
Fixed here [ee84147](https://github.com/OpenFn/adaptors/pull/297/commits/ee84147293f921c322f5fd27f17746e64cb4a458)